client-mvp-04/src/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+@import '../variables';
+
+.dropdown-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  .dropdown-wrapper {
+    display: grid;
+    justify-content: center;
+    .dropdown {
+      display: flex;
+      justify-content: center;
+      width: 230px;
+      border-bottom: 1px solid $dropdown-border-color;
+      margin-bottom: 14px;
+      .selected {
+        width: 100%;
+        min-height: 20px;
+        text-align: left;
+        margin: 0px 0px 6px 3px;
```

Comment:
Please, replace with `margin: 0 0 6px 3px;`
When setting `0`, you don't need `px`.